### PR TITLE
Reenable glasses config

### PR DESCRIPTION
**Pull request by makamys**
As requested by Mitch. I haven't tested with dreamcraft, but I see no reason it wouldn't work.

### CODE REVIEW

**Review by mitchej123 — APPROVED** [review]

**Comment by mitchej123** [comment]
Context - original forcing of glasses 
image

**Comment by mitchej123** [comment]
Also this is mainly for out of pack usage, which I see no reason to restrict.

File: src/main/java/net/dries007/holoInventory/Config.java [hunk 1]
```diff
@@ -21,12 +21,14 @@
...
 
+import cpw.mods.fml.common.Loader;
+
 public class Config {
 
     private static Configuration configuration;
     private final File file;
 
-    public static final boolean requireGlasses = true;
+    public static boolean requireGlasses = Loader.isModLoaded("dreamcraft");
```
Comment (bombcar) [inline src/main/java/net/dries007/holoInventory/Config.java:31]: this is incredibly efficient and l like it heh

File: src/main/java/net/dries007/holoInventory/Config.java [hunk 2]
```diff
@@ -106,8 +108,9 @@ public void doConfig() {
                 .get(HoloInventory.MODID, "renderScaling", renderScaling, "Visual scale factor (0.0-1.0)")
                 .getDouble(1.0);
 
-        // requireGlasses = configuration.get(HoloInventory.MODID, "requireGlasses", requireGlasses,"Makes
-        // HoloInventory require HoloGlasses").getBoolean(true);
+        requireGlasses = configuration
+                .get(HoloInventory.MODID, "requireGlasses", requireGlasses, "Makes HoloInventory require HoloGlasses")
+                .getBoolean(true);
```
Comment (bombcar) [inline src/main/java/net/dries007/holoInventory/Config.java:113]: does this default true even if dream craft is unloaded?

**Review by bombcar — APPROVED** [review]

**Pull request merged**